packages/gatsby/src/query/graphql-errors.js

```diff
@@ -120,11 +134,12 @@ function getCodeFrameFromRelayError(
   return getCodeFrame(query, line, column)
 }
 
-export function multipleRootQueriesError(
-  filePath: string,
-  def: any,
-  otherDef: any
-) {
+/**
+ * @param {string} filePath
+ * @param {any} def
```

Comment:
```suggestion
 * @param {*} def
```
I think `any` isn't a valid JSDoc tag type? https://jsdoc.app/tags-type.html - I might be wrong here, otherwise ignore me

Reply:
Ahh, you are indeed correct. Apologies